frontend/public/components/alert.tsx

```diff
@@ -0,0 +1,469 @@
+/* eslint-disable no-undef */
+
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { coFetchJSON } from '../co-fetch';
+import { alertRuleState } from '../module/monitoring';
+import k8sActions from '../module/k8s/k8s-actions';
+import { MonitoringRoutes, connectToURLs } from '../monitoring';
+import store from '../redux';
+import { UIActions } from '../ui/ui-actions';
+import { ColHead, List, ListHeader, ResourceRow, TextFilter } from './factory';
+import { CheckBoxes } from './row-filter';
+import { SafetyFirst } from './safety-first';
+import { BreadCrumbs, history, NavTitle, SectionHeading, StatusBox, Timestamp, withFallback } from './utils';
+import { formatDuration } from './utils/datetime';
+
+const AlertModel = {
+  kind: 'Alert',
+  label: 'Alert',
+  path: 'alerts',
+  abbr: 'AL',
+};
+export const AlertModelReference = `monitoring:v1:${AlertModel.kind}`;
```

Comment:
"References" are the `GroupVersionKind` representation of a Kubernetes resource, which neither `Alert` nor `AlertRule` are.

Reply:
Removed